src/_includes/layouts/home.liquid

```diff
@@ -1,86 +1,76 @@
-      <h3 class="text-gray mt-5">{{ homeSectionHeader.sectionHeaderSubTitle }}</h3>
-
-      <div class="row">
-        <div class="col-md-4">
-          <h6 class="text-blue mt-5 mb-1">Community over credit</h6>
-          <p>Each win moves us all closer to digital transformation</p>
-
-          <h6 class="text-blue mt-5 mb-1">Purpose over profit</h6>
-          <p>We’re mission-driven, not money-driven</p>
-        </div>
-        <div class="col-md-4">
-          <h6 class="text-blue mt-5 mb-1">Users over stakeholders</h6>
-          <p>If we don’t meet user needs, business goals will never truly be met
-          </p>
+<div class="bg-grayLightest opacity-90 p-4 ">
+  <div class="tablet:mx-10 desktop:px-20 desktop:py-8">
+    <div class="flex flex-col tablet:mx-2 ">
+      <h1 class="text-blue mb-8 pt-6 text-5xl">{{ homeSectionHeader.sectionHeaderTitle }}</h1>
+      <p class="text-gray">{{ homeSectionHeader.sectionHeaderSubText }}</p>
+      <h3 class="text-grayLight my-12 text-3xl">{{ homeSectionHeader.sectionHeaderSubTitle }}</h3>
+    </div>
+    <div class="grid tablet:grid-cols-3 gap-5">
+      <div class="tablet:mx-2">
+        <h6 class="text-blue text-xl">Community over credit</h6>
+        <p class="mb-10">Each win moves us all closer to digital transformation</p>
 
-          <h6 class="text-blue mt-5 mb-1">Innovation over risk management</h6>
-          <p>Change comes from taking chances</p>
-        </div>
-        <div class="col-md-4">
-          <h6 class="text-blue mt-5 mb-1">Outcomes over activities</h6>
-          <p>Success is what you accomplish, not how much you did to get there</p>
+        <h6 class="text-blue text-xl">Purpose over profit</h6>
+        <p class="mb-8">We’re mission-driven, not money-driven</pc>
+      </div>
+      <div class="tablet:mx-2">
+        <h6 class="text-blue text-xl">Users over stakeholders</h6>
+        <p class="mb-10">If we don’t meet user needs, business goals will never truly be met
+        </p>
 
-          <h6 class="text-blue mt-5 mb-1">Delivery over deliberation</h6>
-          <p>Perfection only exists once users tell you it’s perfect</p>
-        </div>
+        <h6 class="text-blue text-xl">Innovation over risk management</h6>
+        <p class="mb-8">Change comes from taking chances</p>
       </div>
-    </div>
-  </div>
+      <div class="tablet:mx-2">
+        <h6 class="text-blue text-xl">Outcomes over activities</h6>
+        <p class="mb-8">Success is what you accomplish, not how much you did to get there</p>
 
-  <div class="bg-parallax">
-    <div>
-      <img src="/assets/img/bg-rectangle-blue.png" />
+        <h6 class="text-blue text-xl">Delivery over deliberation</h6>
+        <p class="mb-10">Perfection only exists once users tell you it’s perfect</p>
+      </div>
     </div>
   </div>
+</div>
 
-</section>
+<div class="w-full h-64 object-cover bg-cover bg-fixed bg-no-repeat shadow-lg" style="background-image: url('/assets/img/bg-rectangle-blue.png');"></div>
```

Comment:
Move `background-image` property to external stylesheet instead of using `style` attribute.